### ISSUE TITLE
configuration of base-ip/ipv4 is flakey at best

### ISSUE THREAD

**Issue by dimbleby** [open-switch/opx-cps#44]
If I attempt to set configuration at `base-ip/ipv4` then:

* the transaction always appears to fail
* sometimes the change I'm making sticks anyway

Eg:

```
# cps_get_oid.py target base-ip/ipv4 name=e101-025-0

Key: 1.22.1441814.1441811.1441808.
base-ip/ipv4/vrf-id = 0
base-ip/ipv4/name = e101-025-0
base-ip/ipv4/forwarding = 1
base-ip/ipv4/ifindex = 36
base-ip/ipv4/enabled = 1

# cps_set_oid.py set base-ip/ipv4 name=e101-025-0 forwarding=0

# cps_get_oid.py target base-ip/ipv4 name=e101-025-0

Key: 1.22.1441814.1441811.1441808.
base-ip/ipv4/vrf-id = 0
base-ip/ipv4/name = e101-025-0
base-ip/ipv4/forwarding = 0
base-ip/ipv4/ifindex = 36
base-ip/ipv4/enabled = 1
```

Note that `cps_set_oid.py` did *not* report success as it does when transactions succeed, although the configuration *was* apparently successfully applied.

Attempting to set other fields (eg `enabled`) fails altogether - which is better, in a sense!

**Comment by Tejaswi-Goel** [comment]
The issues are fixed in OPX 2.2.0 .

**Comment by jeff-yin** [comment]
As Tejaswi said, this particular issue should be resolved with the upcoming 2.2.0 release of OPX
